docs/data/date-pickers/rfc-field.md

```diff
@@ -0,0 +1,426 @@
+
+- `selectedSections`: `FieldSelectedSections`
+
+- `onSelectedSectionsChange`: `(newValue: FieldSelectedSections) => void`
+
+- `unstableFieldRef`: `React.Ref<FieldRef<TSection>>`
+
+  This one may not be needed. If its only used for internal purpose, it can probably be made internal (the picker would pass a ref using a private context instead of a prop).
+
+  Keeping it unstable makes removing it easy.
+
+- `autoFocus`: `boolean`
+
+:::success
+All the props that the picker can pass to the field (validation props, value props, etc...) are read both from the props and from `usePickerContext` so that the field can be used inside a picker built with composition.
+
+That way, users only have to pass the props specific to the field to the `PickerField.*` components:
+
+```tsx
+<Picker.Root manager={manager} {...props} disablePast>
+  <PickerField.Root shouldRespectLeadingZeros>
+    {/** See demo above */}
+  </PickerField.Root>
+  <Popover.Positioner>{/** See picker documentation */}</Popover.Positioner>
+</Picker.Root>
+```
+
+:::
+
+### `PickerField.Content`
+
+It would expend `Field.Control` from `@base-ui-components/react/Field`.
+
+It expects a function as its children, which receives the list of sections to render as a parameter:
+
+```tsx
+<PickerField.Content>
+  {({ sections }) =>
+    sections.map((section) => (
+      <PickerField.Section value={section} key={section.key} />
+    ))
+  }
+</PickerField.Content>
+```
+
+It also renders a hidden input which contains the stringified value and can be used for form submission and testing.
+
+#### Props
+
+- Extends `Field.Control.Props`
+
+- `children`: `(section: InferFieldSection<TValue>) => React.ReactNode`
+
+### `PickerField.Section`
+
+Renders a single section (for instance the year of the hour of the current value).
+
+#### Props
+
+- Extends `React.HTMLAttributes<HTMLSpanElement>`
+- `section`: `InferFieldSection<TValue>` (can be `FieldSection` or `FieldRangeSection`) - **required**.
+
+### `PickerField.SectionContent`
+
+Renders the content of a single section.
+
+#### Props
+
+- Extends `React.HTMLAttributes<HTMLSpanElement>`
+
+### `PickerField.SectionSeparator`
+
+Renders the separator to display before or after the current section.
+
+```tsx
+<PickerField.SectionSeparator position="before" />
+```
+
+#### Props
+
+- Extends `React.HTMLAttributes<HTMLSpanElement>`
+- `position`: `'before' | 'after'` - **required**.
+
+### `PickerField.Clear`
+
+Renders the button to clear the value of the field.
+
+#### Props
+
+- Extends `React.HTMLAttributes<HTMLButtonElement>`
+
+- `children`: `React.ReactNode`
+
+- `onClear`: `React.MouseEventHandler`
+
+  :::success
+  The new DX is a good opportunity to discuss the behavior of this prop.
+  The behavior should either be:
+
+  1. `onClear` is defined on `<PickerField.Root />`. It is also fired when doing <kbd class="key">Ctrl</kbd> + <kbd class="key">A</kbd> and then <kbd class="key">Backspace</kbd>.
```

Comment:
I like the idea of having one `onClear` method defined on the higher level, that is executed every time a clearing action is triggered (click on the button or by keyboard). Makes things more consistent IMO